PopUp.py

```diff
@@ -36,8 +37,8 @@ def __init__(self, title, msg, more=None, yesMes=None, noMes=None,
 
     def questionWindow(self):
         self.setWindowTitle(self.title)
-        self.setGeometry(self.left, self.top, self.width, self.height)
-
+        if not self.parentless:
```

Comment:
not quite - that assumes that all values in setGeometry are inherited. This is not the case for some windows i.e. popup (or previously settings)windows - trying to accesss any of them causes error.
Besides, this might be undefined behaviour if we try to reset window inbuilt geometry.
What I mean exactly is that when we call some window from context of another window, then it is fine, 
because we know parent window will have a context and assigned geometry parameters. But some windows are not called from context, mostly from other non-qt -importing classes thus making them parentless. In that case Pop Up window geometry is set by system default size and does not require change.